client/src/reducers/variant.js

```diff
@@ -1,12 +1,12 @@
 /* eslint-disable  */
 import PropTypes from 'prop-types';
 import { produce } from 'immer';
-import { cloneDeep, isEqual, findIndex, last } from 'lodash';
-
+import { findIndex } from 'lodash';
+import uuidv1 from 'uuid/v1';
 import * as actions from '../actions/type';
 import { normalizePatientDetails } from '../helpers/struct';
-
-const MAX_REVISIONS = 10;
+import { getUpdatedDraftHistory, getPreviousDraft } from '../helpers/draftQueriesHistory';
+import { getUpdatedDraftAddInstruction, getUpdatedDraftRemoveInstruction } from '../helpers/instructions';
```

Comment:
Like previously mentioned, the main goal of these components is to eventually become a standalone library. Logic has been moved outside of the `Query` directory, making it more difficult to bundle this.

Reply:
You had this logic duplicated in two different components.

I simply extracted it in one single helper file.

Reply:
It doesn't belong in the components these helpers are now used in the reducer. Because the state is now the Redux store.